### PR TITLE
Pub Sub: Implement initial incorporation

### PR DESCRIPTION
**Pull request by stalberm**
-- Untested

### CODE REVIEW

**Review by theusaf — APPROVED** [review]

**Comment by theusaf** [comment]
Merge conflict: `pubspec.yaml`

File: lib/main.dart [hunk 1]
```diff
@@ -7,10 +7,22 @@ import 'package:binsight_ai/screens/splash/wifi_page.dart';
...
+import 'package:sqflite/sqflite.dart';
+import 'package:binsight_ai/pub_sub/subscriber.dart';
+import 'package:web_socket_channel/io.dart';
+import 'dart:convert';
 
 void main() async {
   WidgetsFlutterBinding.ensureInitialized();
-  await getDatabaseConnection();
+  final Database database = await getDatabaseConnection();
+  final channel = IOWebSocketChannel.connect('http://54.214.80.15/subscribe');
```
Comment (theusaf) [inline lib/main.dart:18]: Location will actually be here
```suggestion
  final channel = IOWebSocketChannel.connect('http://54.214.80.15/api/model/subscribe');
```

**Review by theusaf — CHANGES_REQUESTED** [review]

**Review by theusaf — CHANGES_REQUESTED** [review]
Bump `databaseVersion` in `database/connection.dart`

**Review by bpugh5 — APPROVED** [review]

**Review by theusaf — APPROVED** [review]

**Comment by kimberlymarkley** [comment]
Code review in meeting

**Pull request merged**